guides/v2.1/cloud/project/project-routes-more-redir.md

```diff
@@ -6,142 +6,127 @@ functional_areas:
+Partial redirects work with any type of route, including routes served directly by the application.
 
 Two keys are available under `redirects`:
 
--  `expires`: Optional, the duration that the redirect is cached.
-    Examples of valid values include `3600s`, `1d`, `2w`, `3m`.
-
--  `paths`: The paths to apply the redirects to.
-
-The following defines each rule under `paths`:
-
--  A key that describes the expression to match against the request path
--  A value object describing both the destination to redirect to with detail on how to handle the redirection.
+- **expires**—Optional, specifies the amount of time to cache the redirect in the browser. Examples of valid values include `3600s`, `1d`, `2w`, `3m`.
 
-The value object is defined with the following keys:
+- **paths**—One or more key-value pairs that specify the configuration for partial-route redirect rules.
 
--  [`to`](#cloud-route-partial-to)
--  [`regexp`](#cloud-route-partial-regexp)
--  [`prefix`](#cloud-route-partial-prefix)
--  [`append_suffix`](#cloud-route-partial-append)
--  [`code`](#cloud-route-partial-code)
+  For each redirect rule, the key is an expression to filter request paths for redirection. The value is an object that specifies the target destination for the redirect and options for processing the redirect.
 
-### `to` {#cloud-route-partial-to}
+  The value object has the following properties:
 
-Required, a partial (`"/destination"` or `"//destination"`) or full URL (`"http://example.com/"`).
+  Property  | Description
+  ----------| -----------
+  `to`      | Required, a partial absolute path, URL with protocol and host, or pattern that specifies the target destination for the redirect rule.
+  `regexp`| Optional, defaults to `false`. Specifies whether the path key should be interpreted as a PCRE regular expression.
+  `prefix` | Specifies whether the redirect applies to both the path and all its children, or just the path itself. Defaults to `true`. This value is not supported if `regexp` is `true`.
+  `append_suffix`| Determines if the suffix is carried over with the redirect. Defaults to `true`. This value is not supported if the `regexp` key is `true` *or* if the `prefix` key is `false`.
+  `code`    | Specifies the HTTP status code. Valid status codes are [`301` (Moved Permanently)](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.2), [`302`](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.3), [`307`](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.8), and [`308`](https://tools.ietf.org/html/rfc7238). Defaults to `302`.
+  `expires`| Optional, specifies the amount of time to cache the redirect in the browser. Defaults to the `expires` value defined directly under the `redirects` key, but at this level you can fine-tune the cache expiration for individual partial redirects.
 
-### `regexp` {#cloud-route-partial-regexp}
+### Examples
```

Comment:
Really happy to see examples organized in a separate way. This helps to keep the reference clean. Nice!